brainstormforce/astra
# AST-2798 - Hide "Cart Click" section from responsive view

### Description

- Hiding "Cart Click" section from responsive view as its not needed for responsive cases.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2978

### Screenshots

https://d.pr/i/AK4uQO

### Types of changes


Fix: Hiding "Cart Click" section from responsive view as its not needed for responsive cases.



### How has this been tested?

- Add cart widget
- Go to Woocommerce Cart settings in customizer.
- Switch to tablet / mobile view.
- The Cart Click section ideally should not be visible for responsive devices.

### Checklist:
